tests/test_service.py

```diff
@@ -87,6 +88,70 @@ def test_finish_launch(self, mock_get, rp_service):
             'name', datetime.now().isoformat())
         assert _get_msg == {'id': 111}
 
+    @mock.patch('reportportal_client.service._get_json',
+                mock.Mock(return_value={'id': 112}))
+    def test_get_launch_info(self, rp_service, monkeypatch):
+        """Test get current launch information.
+
+        :param rp_service:  Pytest fixture that represents ReportPortalService
+                            object with mocked session.
+        :param monkeypatch: Pytest fixture to safely set/delete an attribute
+        """
+        mock_get = mock.Mock(return_value={'id': 112})
+        monkeypatch.setattr(rp_service.session, 'get', mock_get)
+
+        launch_id = rp_service.get_launch_info()
+        mock_get.assert_called_once_with(
+            url='{0}/launch/uuid/{1}'.format(rp_service.base_url_v1,
+                                             rp_service.launch_id),
+            verify=rp_service.verify_ssl)
+        assert launch_id == {'id': 112}
+
+    def test_get_launch_ui_id(self, rp_service, monkeypatch):
+        """Test get launch UI ID.
+
+        :param rp_service:  Pytest fixture that represents ReportPortalService
+                            object with mocked session.
+        :param monkeypatch: Pytest fixture to safely set/delete an attribute
+        """
+        mock_get_launch_info = mock.Mock(return_value={'id': 113})
+        monkeypatch.setattr(rp_service,
+                            'get_launch_info',
+                            mock_get_launch_info)
+        launch_ui_id = rp_service.get_launch_ui_id()
+        mock_get_launch_info.assert_called_once_with()
```

Comment:
I think this check can be omitted.

Reply:
Sure, if you think so.  I can remove it. But it is the key operation of the method.

Could you please help me to understand the unit test policy in this project? I`d like to contribute more and would be great to know.
Should I cover only the function's signatures and return value type/format or key operations as well?

E.g. test for the launch start.
```python
    @mock.patch('reportportal_client.service._get_data')
    def test_start_launch(self, mock_get, rp_service):
        """Test start launch and sending request.

        :param mock_get:   Mocked _get_data() function
        :param rp_service: Pytest fixture
        """
        mock_get.return_value = {'id': 111}
        launch_id = rp_service.start_launch('name', datetime.now().isoformat())
        assert launch_id == 111
```

If I reduce the `start_launch()` to this, the test will pass anyway. But obviously, there is a bug in the method...
 We mock only the last function that is called. 
```python
    def start_launch(self, n, d):
        self.launch_id = _get_id({})
        return self.launch_id
```

Reply:
No unit test policy.